src/filter-image/FilterImage.tsx

```diff
@@ -0,0 +1,61 @@
+import { ImageProps, View, Image as RNImage, StyleSheet } from 'react-native';
+import {
+  Svg,
+  FeColorMatrix,
+  Filter,
+  Image,
+  FeColorMatrixProps,
+} from '../index';
+import { FilterPrimitiveStandardAttributes } from '../elements/filters/FilterPrimitive';
+
+export interface FilterImageProps extends ImageProps {
+  filters: Array<
```

Comment:
Could we extract it to a type so we can also export it later in `index` as a direct type?

Reply:
Done https://github.com/software-mansion/react-native-svg/pull/2316/commits/425a0eb720051041cc84db7f8f60af76fb685dc7